CONTRIBUTING.md

```diff
@@ -17,14 +17,41 @@ In order to build the source code of the .NET APM Agent you need
 
 You can use any IDE that supports .NET Core development and you can use any OS that is supported by .NET Core.
 
-## Code contributions
+## Code contributions (please read this before your first PR)
 
 If you have a bugfix or new feature that you would like to contribute,
 please find or open an issue about it first.
 Talk about what you would like to do.
 It may be that somebody is already working on it,
 or that there are particular issues that you should know about before implementing the change.
 
+We aim to maintain high code quality, therefore every PR goes through the same review process, no matter who opened the PR.
+
+### General advice
+
+Please make sure that your PR addresses a single issue and its size is reasonable to review. There are no hard rules, but please keep in mind that every line of your PR will be reviewed and it's not uncommon that a longer discussion evolves in case of a sensitive change.
+
+Therefore it's preferred to have multiple smaller PRs over a single big PR.
+
+This makes sure that obvious changes get merged quickly and don't get blocked by unrelated changes. Additionally, the repository will also have a useful and searchable history by doing so.
```

Comment:
I'm not sure "obvious changes" is the right term - I would go with something like "changes that have consensus".

Reply:
Changed it accordingly.